app/store/index.ts

```diff
@@ -31,6 +32,18 @@ const persistConfig = {
 
 const persistedReducer = persistReducer(persistConfig, rootReducer);
 
-export const store = createStore(persistedReducer, {}, enhancers);
-StoreAccessor.setStore(store);
-export const persistor = persistStore(store);
+export const createStoreAndPersistor = (
```

Comment:
I don't think this should export the `createStoreAndPersistor` since this is meant to be a one time operation, given the `StoreAccessor`, I think we should stick to exporting the singleton

Reply:
I think this makes way more sense, thanks! I took a hack at moving the initialization to this class and pass config to it, right now is just a flag but it could become an object with a config if it gets hairier than the one middleware.